### PR TITLE
OpenCVE CVSS Score

### PR DESCRIPTION
**Pull request by anas-yousef**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-36994

## Description
Added a way to extract the cvss score from the API response.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/OpenCVE/ReleaseNotes/1_0_4.md
Comment (JasBeilin) [inline Packs/OpenCVE/ReleaseNotes/1_0_4.md:1]: Lets be clear on which value is being taken.

**Review by JasBeilin — CHANGES_REQUESTED** [review]
Looks great.

File: Packs/OpenCVE/ReleaseNotes/1_0_4.md
Comment (anas-yousef) [reply Packs/OpenCVE/ReleaseNotes/1_0_4.md:1]: Better?

**Comment by anas-yousef** [comment]
@JasBeilin I added your suggestions.

File: Packs/OpenCVE/ReleaseNotes/1_0_4.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### OpenCVE
+
+- Updated the Docker image to: *demisto/python3:3.10.14.92207*.
+- Added a fix where the CVSS score was sometimes not extracted if found under the **metrics** key. It will be extracted from the **cvss** key in such a case.
```
Comment (JasBeilin) [inline Packs/OpenCVE/ReleaseNotes/1_0_4.md:7]:
```suggestion
- CVSS score will be extracted from the **cvss** key, insead of **metrics** key.
```

**Review by JasBeilin — APPROVED** [review]

**Pull request merged**